.codegen/service.py.tmpl

```diff
@@ -13,7 +13,7 @@ from ._internal import _enum, _from_dict, _repeated, Wait
 _LOG = logging.getLogger('databricks.sdk')
 
 {{range .ImportedEntities}}
-from .{{.Package.Name}} import {{.PascalName}}{{end}}
+from . import {{.Package.Name}}{{end}}
```

Comment:
this will break `catalog`, `sharing`, and `jobs` with external imports, please revert

Reply:
ah, i see - `'{{.Package.Name}}.{{.PascalName}}'`

can you change dot import to `from databricks.sdk.service import {{.Package.Name}}`

Reply:
Done.